### PR TITLE
Pull to refresh is here

### PR DESCRIPTION
**Pull request by nirzar**
- Added refresh to scrollview of webview
- Hooked up original refresh action
- Removed the refresh from toolbar

NOTE: refresh button was never meant for article toolbar. this is almost a regression issue over the app store version.

### CODE REVIEW

File: Wikipedia/Code/WMFArticleViewController.m [hunk 1]
```diff
@@ -705,6 +710,7 @@ - (void)fetchArticleForce:(BOOL)force {
 
 - (void)fetchArticle {
     [self fetchArticleForce:YES];
+    [_pullToRefresh endRefreshing];
```
Comment (bgerstle) [inline Wikipedia/Code/WMFArticleViewController.m]: shouldn't refresh stay visible until article finishes being downloaded?

File: Wikipedia/Code/WMFArticleViewController.h [hunk 2]
```diff
@@ -32,6 +32,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, strong, readonly) MWKTitle* articleTitle;
 @property (nonatomic, strong, readonly) MWKDataStore* dataStore;
+@property (nonatomic, strong, readonly) UIRefreshControl* pullToRefresh;
```
Comment (bgerstle) [inline Wikipedia/Code/WMFArticleViewController.h]: need to remove `refreshToolbarItem` getter method as well

File: Wikipedia/Code/WMFArticleViewController.m (hunk 1 above)
Comment (nirzar) [inline Wikipedia/Code/WMFArticleViewController.m]: When the refresh is ended the progress bar takes over and hence there is no need for showing the  activity indicator.

**Comment by coreyfloyd** [comment]
:+1:

**Pull request merged**